docs/topics/ref_new-features-7-1-0.adoc

```diff
@@ -32,7 +32,7 @@ In {ProductFullName} 7.1.0, you can use {ProductShortName} to analyze Windows-on
 
 [IMPORTANT]
 ====
-Support for analyzing .NET applications is Developer Preview feature only. Developer Preview feature is not supported by Red{nbsp}Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview feature for production or business-critical workloads. Developer Preview feature provides early access to upcoming product feature in advance of its possible inclusion in a Red{nbsp}Hat product offering. Customers can use this feature to test functionality and provide feedback during the development process. This feature might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red{nbsp}Hat might provide ways to submit feedback on Developer Preview feature without an associated SLA.
+Support for analyzing .NET applications is a Developer Preview feature only. Developer Preview features are not supported by Red Hat in any way and are not functionally complete or production-ready. Do not use Developer Preview features for production or business-critical workloads. Developer Preview features provide early access to upcoming product features in advance of their possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. These features might not have any documentation, are subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview features without an associated SLA.
```

Comment:
See the snippets PR which will ensure uniformity in language

* [MTA-3529](https://issues.redhat.com/browse/MTA-3529)

Please can you and Radhika approve and merge so we can use this snippet